src/helper/ssg/index.test.tsx

```diff
@@ -30,20 +30,28 @@ describe('toSSG function', () => {
     app.get('/Charlie', (c) => {
       return c.render('Hello!', { title: 'Charlies Page' })
     })
+    // Included params
+    app.get('/post/:post', ssgParams(() => [
+      {
+        post: '1'
+      },
+      {
+        post: '2'
+      }
+    ]), c => c.html(<h1>{c.req.param('post')}</h1>))
 
     const fsMock = {
       writeFile: vi.fn((path, data) => Promise.resolve()),
       mkdir: vi.fn((path, options) => Promise.resolve()),
     }
 
-    const htmlMap = await generateHtmlMap(app)
+    const htmlMap = await generateHtmlMap(app, {})
```

Comment:
We should write the tests for checking the contents of `htmlMap`. Like this:

```ts
expect(htmlMap.get('/post/1')).toBe('<h1>1</h1>')
```

Reply:
I did it.